app/Models/Thread.php

```diff
@@ -68,6 +68,13 @@ final class Thread extends Model implements ReplyAble, SubscriptionAble, Feedabl
         'updatedByRelation',
     ];
 
+    /**
+     * {@inheritdoc}
+     */
+    protected $dates = [
+        'last_activity_at',
+    ];
+
```

Comment:
The `$dates` property is actually deprecated and should be avoided. Try using `$casts` instead.